app/services/nebraska_full_days_calculator.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+# Service to calculate full days used in Nebraska by specific kids
+class NebraskaFullDaysCalculator
+  def initialize(child, filter_date)
+    @child = child
+    @filter_date = filter_date
+  end
+
+  def call
+    calculate_full_days
+  end
+
+  private
+
+  def calculate_full_days
+    @child.attendances.for_month(@filter_date).reduce(0) do |sum, attendance|
+      sum + calculate_full_days_based_on_duration(attendance.total_time_in_care)
+    end
+  end
+
+  def calculate_full_days_based_on_duration(duration)
+    if duration >= (14.hours + 45.minutes)
+      2
+    elsif duration >= (5.hours + 45.minutes)
```

Comment:
To discuss: Right now we have this 5.hours + 45.minutes parameter in two places (here + `hours_calculator`).  From a DRY perspective, should these services be combined to return two outputs (hours and days)?  the child model could pull the correct output in `nebraska_hours` and `nebraska_full_days` respectively.

Reply:
Yeah this is a good point, I like this direction.  Let me see if I can pull that logic out.

Reply:
On further investigation I want to refactor our service objects into procedural programming so I'm gonna skip this for now.